scripts/deployment/hf_upload/config_update.py

```
@@ -0,0 +1,67 @@
+import logging
+from typing import Any, Dict
+from string import Template
+
+
+def _read(path: str) -> str:
+    with open(path, 'r') as f:
+        return f.read()
+
+
+def _write(path: str, content: str) -> None:
+    os.makedirs(os.path.dirname(path), exist_ok=True)
```

Ruff (0.12.2)

`os.makedirs()` should be replaced by `Path.mkdir(parents=True)`

(PTH103)